sdk/keyvault/azure-security-keyvault-administration/README.md

```diff
@@ -130,6 +130,137 @@ To update the value of any of the the available settings, we will call the Updat
 Setting updatedSetting
    = settingsClient.UpdateSetting(settingsList.Value[0].Name, options).Value;
 ```
+## Creating a BackupRestoreClient
+
+To create a new `BackupRestoreClient` to perform these operations, you need the endpoint to an Azure Key Vault HSM and credentials.
+
+Key Vault BackupRestoreClient client for C++ currently supports any `TokenCredential` for authenticating.
+
+```cpp Snippet:SampleBackupRestoreCreateCredential
+  auto credential
+      = std::make_shared<Azure::Identity::DefaultAzureCredential>();
+```
+
+Then, in the sample below, you can set `keyVaultUrl` based on an environment variable, configuration setting, or any way that works for your application.
+
+```cpp Snippet:SampleAdministration2CreateClient
+  // create client
+  BackupRestoreClient client(std::getenv("AZURE_KEYVAULT_HSM_URL"), credential);
+```
+## Create the SasTokenParameter
+
+Since these operations require a blob storage for the backup/restore operations, a SAS token is required for the connection between the services(Key Vault and Storage).  
+
+In this sample we rely on a couple of extra environment variables. 
+
+```cpp Snippet:SampleAdministration2CreateSASParam
+ SasTokenParameter sasTokenParameter;
+ // the backup/restore needs a SAS token to access the storage account
+ sasTokenParameter.Token
+     = Azure::Core::_internal::Environment::GetVariable("AZURE_KEYVAULT_BACKUP_TOKEN");
+ // the backup/restore needs a url to a blob storage resource
+ sasTokenParameter.StorageResourceUri
+     = Azure::Core::_internal::Environment::GetVariable("AZURE_KEYVAULT_BACKUP_URL");
+```
+
+## The Backup operation 
+
+Since this is a long running operation the service provides endpoints to determine the status while the opperation is running. 
+
+### Starting the backup operation
+
+```cpp Snippet:SampleAdministration2StartBackup
+// Create a full backup using a user-provided SAS token to an Azure blob storage container.
+auto backupResponse = client.FullBackup(sasTokenParameter);
```

Comment:
i was wondering the same thing, but .Net does the same thing with implementation and i used that as a template

Reply:
According to .Net folks, this cod is auto generated, and following the swagger, and it might not have been recognized as a LRO. since we also generated the code using the codegen we might have fallen into the same issue.